src/pages/cli/usage/lambda-triggers.mdx

```diff
@@ -91,8 +95,15 @@ import React from 'react';
 import './App.css';
 import { Amplify, Auth, Hub } from 'aws-amplify';
 import awsconfig from './aws-exports';
-import { Authenticator, SignUp, SignIn, Greetings, ConfirmSignUp, AuthPiece } from 'aws-amplify-react';
-import ReCAPTCHA from "react-google-recaptcha";
+import {
+  Authenticator,
+  SignUp,
+  SignIn,
+  Greetings,
+  ConfirmSignUp,
+  AuthPiece
+} from 'aws-amplify-react';
```

Comment:
I think this import is from the old authenticator package - shouldn't it be @aws-amplify/ui-react